code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -159,6 +159,25 @@
 	if(delold)
 		qdel(src)
 
+/mob/living/silicon/pai/attackby(obj/item/W as obj, mob/user as mob, params)
+	if(istype(W, /obj/item/stack/nanopaste))
+		var/obj/item/stack/nanopaste/N = W
+		if(stat == DEAD)
+			to_chat(user, "<span class='danger'>[src] is beyond help, at this point.</span>")
+		else if(getBruteLoss() || getFireLoss())
+			adjustBruteLoss(-15)
+			adjustFireLoss(-15)
+			updatehealth()

```

Comment:
pretty sure both of the two adjustXLoss procs above call updatehealth? Is this still needed?

Reply:
It is not. Remov!!11!111!!11!!!!!